docs/included.md

```diff
@@ -264,13 +304,37 @@
   - 
     [Hanssen and Kuipers discriminant (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#HK)
 * -  
-    - Hit Rate (True Positive Rate, Probability of Detection (POD), Sensitivity)
+    - Heidke Skill Score (Cohen's Kappa)
+  -
+    [API](https://scores.readthedocs.io/en/latest/api.html#scores.categorical.BasicContingencyManager.heidke_skill_score)
+  - 
+    [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Binary_Contingency_Scores.html)
+  - 
+     [Threat score (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#HSS) 
+* -  
+    - Hit Rate (True Positive Rate, Probability of Detection (POD), Sensitivity, Recall)
   -
     [API](https://scores.readthedocs.io/en/latest/api.html#scores.categorical.BasicContingencyManager.hit_rate)
   - 
     [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Binary_Contingency_Scores.html)
   - 
     [Probability of detection (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#POD)
+* -  
+    - Odds Ratio
+  -
+    [API](https://scores.readthedocs.io/en/latest/api.html#scores.categorical.BasicContingencyManager.odds_ratio)
+  - 
+    [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Binary_Contingency_Scores.html)
+  - 
+    [Stephenson (2000)](https://journals.ametsoc.org/view/journals/wefo/15/2/1520-0434_2000_015_0221_uotorf_2_0_co_2.xml?tab_body=fulltext-display)
```

Comment:
```suggestion
    [Stephenson (2000)](https://doi.org/10.1175/1520-0434(2000)015%3C0221:UOTORF%3E2.0.CO;2)
```

Reply:
@tennlee has previously said he thinks moving to DOIs where available is probably worthwhile, given the additional stability of DOIs. 

I know we haven't changed all the links as yet, but I have been opportunistically doing so/suggesting to do so when I notice them and it is easy to do so.